experimental/origin-dapp2/src/css/app.css

```diff
@@ -23,7 +23,7 @@ body {
 }
 
 .loading-screen {
-  background: url(images/spinner-animation-dark.svg) no-repeat center;
+  /* background: url('images/spinner-animation-dark.svg') no-repeat center;*/
```

Comment:
remove ?

Reply:
I had to comment this out because it was (weirdly) causing some build errors. It doesn't anymore so thanks removing this comment